package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "author": "xh3b4sd",
+  "bugs": {
+    "url": "https://github.com/venturemark/numnum/issues"
+  },
+  "dependencies": {
+    "google-protobuf": "^3.13.0",
+    "grpc-web": "^1.2.1"
+  },
+  "description": "number detection from string bodies",
+  "devDependencies": {
+    "@rollup/plugin-commonjs": "^16.0.0",
+    "@rollup/plugin-typescript": "^6.1.0",
+    "@types/jest": "^26.0.15",
+    "jest": "^26.6.3",
+    "rollup": "^2.32.1",
+    "rollup-plugin-terser": "^7.0.2",
+    "ts-jest": "^26.4.4",
+    "tslib": "^2.0.3",
+    "typescript": "^4.1.2"
+  },
+  "exports": {
+    ".": {
+      "default": "./dst/index.es.js",
+      "require": "./dst/index.js"
+    }
+  },
+  "files": [
+    "dst"
+  ],
+  "homepage": "https://github.com/venturemark/numnum#readme",
+  "license": "MIT",
+  "main": "./dst/index.js",
+  "module": "./dst/index.es.js",
+  "name": "@venturemark/numnum",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/venturemark/numnum.git"
+  },
+  "scripts": {
+    "build": "./node_modules/rollup/dist/bin/rollup -c --exports named",
+    "test": "./node_modules/jest/bin/jest.js"
```

Comment:
Note there is no local prerequisite required to build the project or run tests. This is about simplicity for the user. This is about clean interfaces. Nobody has to worry about the black magic inside the box.